src/index.js

```diff
@@ -242,20 +252,98 @@ const urbitKeysFromSeed = (seed, password) => {
 
 
 
+/**
+ * Reduce a collection of arrays by recursive applications of bytewise XOR.
+ * @param  {array of array of integers}  arrays an array of arrays
+ * @return {array} the resulting array
+ */
+const reduceByXor = (arrays) => {
+  return arrays.reduce((acc, arr) =>
+    lodash.zipWith(acc, arr, (x, y) => x ^ y));
+}
+
+
+
+/**
+ * Shard a hex-encoded string into three shards, such that any two shards can
+ * be combined to recover it.
+ * @param  {string}  string hex-encoded string
+ * @return {array of strings} resulting shards
+ */
+const shard = (hex) => {
+  const buffer = hex2buf(hex);
+  const sharded = shardBuffer(buffer);
+  return sharded.map(pair =>
+           pair.map(arr => buf2hex(bufferFrom(arr))));
```

Comment:
So a shard is a pair of two hex strings. Should we concat those in some way for ease of use?